src/js/index.js

```diff
@@ -25,49 +25,84 @@ menuItems.forEach(function (menuItem) {
 })
 
 function resetMenuClasses(items) {
-  items.forEach(function(item) {
+  items.forEach(function (item) {
     item.classList.remove(menuOpenClass);
   })
 }
 
 // change icon menu mob list
-menuMobItems.forEach(function(menuMobItem){
-  menuMobItem.addEventListener('click', function(event) {
-      event.stopPropagation();
-      var clickedItemMob = event.currentTarget;
-      var isOpenMob = clickedItemMob.classList.contains(menuMobOpenClass);
-      resetMenuMobClasses(menuMobItems);
+menuMobItems.forEach(function (menuMobItem) {
+  menuMobItem.addEventListener('click', function (event) {
+    event.stopPropagation();
+    var clickedItemMob = event.currentTarget;
+    var isOpenMob = clickedItemMob.classList.contains(menuMobOpenClass);
+    resetMenuMobClasses(menuMobItems);
 
-      if (!isOpenMob) {
-          clickedItemMob.classList.add(menuMobOpenClass);
-      }
+    if (!isOpenMob) {
+      clickedItemMob.classList.add(menuMobOpenClass);
+    }
   })
 })
 
 function resetMenuMobClasses(items) {
-  items.forEach(function(item) {
+  items.forEach(function (item) {
     item.classList.remove(menuMobOpenClass);
   })
 }
 
 // click on the bottom menu-mob and opet window
-document.body.addEventListener('click', function() {
+document.body.addEventListener('click', function () {
   resetMenuClasses(menuItems);
-}) 
+})
 
 var menuMobile = document.querySelector('.menu-mobile');
 var menuBtn = document.querySelector('.menu-btn');
 
-menuBtn.addEventListener('click', function() {
+menuBtn.addEventListener('click', function () {
   menuMobile.classList.toggle('menu-mobile__open');
 })
 
-$(document).ready(function(){
+$(document).ready(function () {
   $('.carousel').slick({
     infinite: true,
     slidesToShow: 1,
     slidesToScroll: 1,
     dots: false,
     adaptiveHeight: true
   });
+});
+
+$(document).ready(function () {
```

Comment:
You usually do `$(document).readt(function () {})` just once.

Please put all the code inside a single `document ready` function:

```diff
- $(document).ready(function () {
-   foo()
- })
-
- $(document).ready(function () {
-   bar()
- })
+ $(document).ready(function () {
+   foo()
+   bar() // together
})
```